src/SwedbankPay.Sdk/Payments/SwishPayments/Operations.cs

```diff
@@ -2,14 +2,14 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace SwedbankPay.Sdk.Payments.Swish
+namespace SwedbankPay.Sdk.Payments.SwishPayments
 {
     public class Operations : Dictionary<LinkRelation, HttpOperation>
     {
         public Func<Task<PaymentResponse>> Abort { get; internal set; }
         public Func<ReversalRequest, Task<ReversalResponse>> CreateReversal { get; internal set; }
         public Func<SaleRequest, Task<SaleResponse>> CreateSale { get; internal set; }
-        public HttpOperation this[LinkRelation rel] => ContainsKey(rel) ? base[rel] : null;
+        public new HttpOperation this[LinkRelation rel] => ContainsKey(rel) ? base[rel] : null;
```

Comment:
`new` is a bit dirty. Can we just implement `IDictionary` instead of inheriting `Dictionary`?

Reply:
Sure, implementing `IDictionary` is 15 methods.

Reply:
Please add a base class that implements the index accessor and `IDictionary` so we can get rid of the `new`.

Reply:
Can do